### PR TITLE
Update peril.settings.json

### PR DESCRIPTION
**Pull request by ashkan18**
# Problem
Implementing slack notification for issues created in https://github.com/artsy/README/issues/172

### CODE REVIEW

File: org/retroActionItems.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { peril } from "danger"
+import { Issues } from "github-webhook-event-types"
+import { IncomingWebhookSendArguments, MessageAttachment } from "@slack/client"
+
+/**
+ * When an issue has been labelled RFC, then trigger the scheduler
+ * to send reminders about the issue into our slack.
+ */
```
Comment (orta) [inline org/retroActionItems.ts]: Comment needs updating, ideally with a link to your RFC?

Comment (orta) [reply org/retroActionItems.ts]: Good to go after ^

Comment (ashkan18) [reply org/retroActionItems.ts]: 🙌 merging on green then

**Comment by ashkan18** [comment]
merge on green

**Pull request merged**